src/Microsoft.ML.TorchSharp/NasBert/TextClassificationTrainer.cs

```diff
@@ -812,27 +842,83 @@ private sealed class Mapper : MapperBase
                         _inputColIndices.Add(col);
 
                 _labelCol = inputSchema[_parent._options.LabelColumnName];
+                _inputSchema = inputSchema;
+
+                torch.random.manual_seed(1);
```

Comment:
>torch [](http://example.com/codeflow?start=16&length=5)

could you please add a comment why we are setting the seeds here?

Also, in toch is it possible to disable the random seed all together? or this will give undesired results?